include/seahorn/seahorn.h

```diff
@@ -47,7 +47,8 @@ extern void sea_tracking_on(void);
 extern void sea_tracking_off(void);
 /* reset modified metadata for memory pointed to by arg */
 extern void sea_reset_modified(char *);
-
+extern void sea_set_shadowmem(char, char *, char);
```

Comment:
need a brief comment description for `sea_set/get_shadowmem`

Reply:
done